lib/protobuf/message.rb

```diff
@@ -77,14 +77,13 @@ def each_field
       return to_enum(:each_field) unless block_given?
 
       self.class.all_fields.each do |field|
-        value = __send__(field.getter)
-        yield(field, value)
+        yield(field, __send__(field.getter))
```

Comment:
After #302 it would be the style to use `yield(field, self[field.name])`, you'll probably get a merge conflict anyway, and in `each_field_for_serialization` it would probably be `self[field.name]` too.